ajax/task.php

```diff
@@ -110,6 +110,9 @@
     }
 }
 
+if ($template->fields['users_id_tech'] == 0) {
+    $template->fields['users_id_tech'] = Session::getLoginUserID();
```

Comment:
If the previously selected tech is not the current user, I guess it will be replaced. This will still be a wrong behaviour.

Reply:
You're right - an unset is better